src/utils/nfts.ts

```diff
@@ -29,17 +34,34 @@ export const findLastAction = (nft: NFTParams) => {
 export const isTokenId = (id: any) => {
   const parsedId = Number(id);
 
-  return typeof id !== 'undefined' && !isNaN(parsedId) && parsedId >= 0;
-}
+  return (
+    typeof id !== 'undefined' && !isNaN(parsedId) && parsedId >= 0
+  );
+};
 
 export const getTokenMetadataThumbnail = ({
   tokendId,
   tokenMetadataById,
 }: {
-  tokendId: string | number,
-  tokenMetadataById: TokenMetadataById,
+  tokendId: string | number;
+  tokenMetadataById: TokenMetadataById;
 }) => typeof tokendId !== 'undefined' && tokenMetadataById[tokendId];
 
-export const roundOffDecimalValue = (value: number, decimalPlace: number) => {
+export const roundOffDecimalValue = (
+  value: number,
+  decimalPlace: number,
+) => {
   return value.toFixed(decimalPlace);
-}
\ No newline at end of file
+};
+
+export const isNFTOperatorIsMKP = (
```

Comment:
Seems that we don't need two "is" in the name; as such, we can have a more meaningful name e.g. `isOperatorMarketplace`

```suggestion
export const isOperatorMarketplace = (
```

Reply:
Yeah updated. Thanks!